Loop54.Shared/Model/Request/GetRelatedEntitiesRequest.cs

```diff
@@ -36,5 +36,16 @@ public GetRelatedEntitiesRequest(Entity entity)
         /// Parameters for specifying which results to retrieve. Such as filtering, faceting, sorting and paging. 
         /// </summary>
         public EntityCollectionParameters ResultsOptions { get; set; } = new EntityCollectionParameters();
+        
+        /// <summary>
+        /// The kind of relation that will be used to create resulting entities. Similar or Complementary.
+        /// Defaults to Similar in the engine if not specified.
+        /// </summary>
+        /// <remarks>
+        /// When getting related entities for 'BrandA bike tire': <br/>
+        /// "similar" - will return entities that are 'more of the same' (f.e. 'BrandB bike tire')<br/>
```

Comment:
Good examples! These are already documented on the RelationKinds enum, so I think it would be good to move the examples there, so everything is one place.

Also "that are 'people also buy'" could just be "that people also buy"